cmd/context/selector.go

```diff
@@ -29,6 +29,7 @@ import (
 	"github.com/manifoldco/promptui"
 	"github.com/manifoldco/promptui/list"
 	"github.com/manifoldco/promptui/screenbuf"
+	"github.com/okteto/okteto/pkg/errors"
```

Comment:
Can we rename this import to oktetoErrors so that we keep open the default import if/when needed?

Reply:
fixed at https://github.com/okteto/okteto/pull/2090/commits/20e2c39a321e3888280cf4a1228413e68a8fa58b